src/styles/styles.js

```diff
@@ -3779,6 +3779,13 @@ const styles = (theme) => ({
         lineHeight: variables.lineHeightLarge,
     },
 
+    aspectRatioLottie: (source) => {
+        if (typeof source === 'object' && !source.uri) {
```

Comment:
We can add more safety condition here, though all Lottie JSON may include `w`, `h` properties.
```suggestion
        if (!source.uri && typeof source === 'object' && source.w && source.h) {
```